src/pages/careers/locations/_location-layout.js

```diff
@@ -59,10 +62,11 @@ const Hero = ({ display_name, img_data, description }) => {
                     rel="noopener noreferrer"
                     external
                 >
-                    {`View open positions in ${display_name}`}
+                    <Localize translate_text="View open positions in " />
+                    {display_name}
```

Comment:
This should be wrapped up with `Localize`

`<Localize ...props> {display_name} </Localize>
`